Add details about the new geomap widget (#9526)
DataDog/documentation: content/en/dashboards/widgets/geomap.md

---
title: Geomap Widget
kind: documentation

> widget_type: "geomap"

Reviewer: this has to map to the api spec - https://github.com/DataDog/datadog-api-spec/blob/master/spec/v1/dashboard.yaml - which doesn't look like it includes this one yet. Do you know if there's a plan to add it?

Author: I'm pinging the team to add it

> widget_type: "geomap"

Reviewer: ```suggestion
```

aliases:
    - /graphing/widgets/geomap/
further_reading:
- link: "/dashboards/timeboards/"
  tag: "Documentation"
  text: "Timeboards"
- link: "/dashboards/screenboards/"
  tag: "Documentation"
  text: "Screenboards"
- link: "/dashboards/graphing_json/"
  tag: "Documentation"
  text: "Building Dashboard using JSON"
- link: "/notebooks/"
  tag: "Documentation"
  text: "Notebooks"
---

> The geomap widget graphs any metric that has a across country tag/facet.

Reviewer: ```suggestion
The geomap widget graphs any metric that has an across country tag or facet.
```

{{< img src="dashboards/widgets/geomap/geomap.png" alt="Geomap" >}}

## Setup

{{< img src="dashboards/widgets/geomap/geomap_setup.png" alt="Top List"  style="width:80%;">}}

### Configuration

1. Choose the data to graph:
    * RUM: See [the RUM documentation][2] to configure a RUM query. 
    * Log Events: See [the log search documentation][3] to configure a log event query.

>       * Note: The group by tag must include a country ISO Code. You can use the [GeoIP Processor][6] to do this, or manually include the [tags on ingest][8].

Reviewer: ```suggestion
      * **Note**: The group by tag must include a country ISO Code. You can use the [GeoIP Processor][6] to do this, or manually include the [tags on ingest][8].
```

    * Metric: See the [querying][1] documentation to configure a metric query. 

>       * Note: The group by tag must include a country ISO Code. You can [Generate Metrics from Injested Logs][7], or manually include the [tags on ingest][8].

Reviewer: ```suggestion
      * **Note**: The group by tag must include a country ISO Code. You can [generate metrics from ingested logs][7], or manually include the [tags on ingest][8].
```

2. Optional: configure your view box depending on where you'd like to zoom into on the map by defailt.

### Options

#### Global time

On screenboards and notebooks, choose whether your widget has a custom timeframe or uses the global timeframe.

#### Title

Display a custom title for your widget by activating the `Show a Title` check box:

{{< img src="dashboards/widgets/options/title.png" alt="Widget title"  style="width:80%;">}}

Optionally define its size and alignment.

> ## API
> 
> This widget can be used with the **Dashboards API**. Refer to the [Dashboards API][4] documentation for additional reference.
> 
> The dedicated [widget JSON schema definition][5] for the top list widget is:
> 
> {{< dashboards-widgets-api >}}

Reviewer: ```suggestion
```
